src/common/models/advertisementModels.ts

```diff
@@ -0,0 +1,13 @@
+export class Advertisement {
+  constructor(props = {}) {
+      Object.assign(this, props);
+  }
+
+  id!: number;
```

Comment:
Will be `string` in firebase.

Please also change in other models

Reply:
changed